{"uid":"e3c829b7e5bd21d1","name":"Checks remote card includes status","fullName":"Checks remote card includes status","historyId":"02e0bbfe1a6298159eacb8b3246dfdfa","time":{"start":1681666581677,"stop":1681666583388,"duration":1711},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","steps":[{"name":"visit http://localhost:4173","time":{"start":1681666581838,"stop":1681666581880,"duration":42},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"waitUntil {}","time":{"start":1681666581880,"stop":1681666583388,"duration":1508},"status":"passed","steps":[{"name":"uncaught exception TypeError: ReactDOM.createRoot is not a function","time":{"start":1681666581882,"stop":1681666583388,"duration":1506},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"get [data-e2e=\"APP__CARD\"]","time":{"start":1681666583387,"stop":1681666583387,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"assert expected **undefined** to have a length of **2** but got **0**","time":{"start":1681666583387,"stop":1681666583387,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583387,"stop":1681666583387,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583387,"stop":1681666583387,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681666583388,"stop":1681666583388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":18,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[{"uid":"fb38df8609672a5b","name":"e2e/tests/runAll.cy.ts:2023-04-16T17:36:22.976Z:attempt-1","source":"fb38df8609672a5b.png","type":"image/png","size":149921},{"uid":"3da7591c4f6a0178","name":"video recording","source":"3da7591c4f6a0178.mp4","type":"video/mp4","size":3489871}],"parameters":[],"stepsCount":20,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"e2e.tests.runAll.cy.ts"},{"name":"parentSuite","value":"Vite React Microfrontends"},{"name":"suite","value":"It checks names and symbols inside cards"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"1cc8d64c6d0bf8a4","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681666580576,"stop":1681666581674,"duration":1098}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":33,"broken":0,"skipped":0,"passed":111,"unknown":0,"total":144},"items":[{"uid":"41bec79cb7a349b7","reportUrl":"https://module-federation.github.io/module-federation-examples/710//#testresult/41bec79cb7a349b7","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681596553052,"stop":1681596554921,"duration":1869}},{"uid":"b29ef3fb0b2413d2","reportUrl":"https://module-federation.github.io/module-federation-examples/707//#testresult/b29ef3fb0b2413d2","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681537420732,"stop":1681537422619,"duration":1887}},{"uid":"5a5c3a48bab8aaab","reportUrl":"https://module-federation.github.io/module-federation-examples/704//#testresult/5a5c3a48bab8aaab","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681519391148,"stop":1681519393150,"duration":2002}},{"uid":"fa4a25c75f38cb50","reportUrl":"https://module-federation.github.io/module-federation-examples/696//#testresult/fa4a25c75f38cb50","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681338423291,"stop":1681338425229,"duration":1938}},{"uid":"eb143aab12408ee8","reportUrl":"https://module-federation.github.io/module-federation-examples/695//#testresult/eb143aab12408ee8","status":"passed","time":{"start":1681324165878,"stop":1681324166776,"duration":898}},{"uid":"2b79dde44521bd7b","reportUrl":"https://module-federation.github.io/module-federation-examples/694//#testresult/2b79dde44521bd7b","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681323480438,"stop":1681323482701,"duration":2263}},{"uid":"422b6521a30adfff","reportUrl":"https://module-federation.github.io/module-federation-examples/693//#testresult/422b6521a30adfff","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681299827756,"stop":1681299829870,"duration":2114}},{"uid":"c60e2082e7a9b6ea","reportUrl":"https://module-federation.github.io/module-federation-examples/687//#testresult/c60e2082e7a9b6ea","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681214595665,"stop":1681214597693,"duration":2028}},{"uid":"1ef02bc530545b86","reportUrl":"https://module-federation.github.io/module-federation-examples/686//#testresult/1ef02bc530545b86","status":"passed","time":{"start":1681146978648,"stop":1681146979787,"duration":1139}},{"uid":"251d6bf09c28633","reportUrl":"https://module-federation.github.io/module-federation-examples/685//#testresult/251d6bf09c28633","status":"passed","time":{"start":1681146978648,"stop":1681146979787,"duration":1139}},{"uid":"2d6d291a4efcce1a","reportUrl":"https://module-federation.github.io/module-federation-examples/684//#testresult/2d6d291a4efcce1a","status":"passed","time":{"start":1681136337701,"stop":1681136338426,"duration":725}},{"uid":"5773ba25e1c64f1f","reportUrl":"https://module-federation.github.io/module-federation-examples/683//#testresult/5773ba25e1c64f1f","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681099524058,"stop":1681099526160,"duration":2102}},{"uid":"f43d013de5dafc76","reportUrl":"https://module-federation.github.io/module-federation-examples/682//#testresult/f43d013de5dafc76","status":"passed","time":{"start":1681098145506,"stop":1681098146326,"duration":820}},{"uid":"d3d1a3c6a2a29fe8","reportUrl":"https://module-federation.github.io/module-federation-examples/671//#testresult/d3d1a3c6a2a29fe8","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680987671840,"stop":1680987673760,"duration":1920}},{"uid":"78e1d16f42d9de62","reportUrl":"https://module-federation.github.io/module-federation-examples/665//#testresult/78e1d16f42d9de62","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680909818483,"stop":1680909820330,"duration":1847}},{"uid":"dcbc027d691fd4d8","reportUrl":"https://module-federation.github.io/module-federation-examples/661//#testresult/dcbc027d691fd4d8","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680834545207,"stop":1680834547288,"duration":2081}},{"uid":"43496295717f922b","reportUrl":"https://module-federation.github.io/module-federation-examples/657//#testresult/43496295717f922b","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680806560163,"stop":1680806562716,"duration":2553}},{"uid":"4d5f724a6a3e7b1d","reportUrl":"https://module-federation.github.io/module-federation-examples/630//#testresult/4d5f724a6a3e7b1d","status":"passed","time":{"start":1680739153248,"stop":1680739154187,"duration":939}},{"uid":"c44e849e3bf32b58","reportUrl":"https://module-federation.github.io/module-federation-examples/619//#testresult/c44e849e3bf32b58","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680701570185,"stop":1680701572275,"duration":2090}},{"uid":"511e2e4000cecb7d","reportUrl":"https://module-federation.github.io/module-federation-examples/616//#testresult/511e2e4000cecb7d","status":"passed","time":{"start":1680695677736,"stop":1680695678660,"duration":924}}]},"tags":[]},"source":"e3c829b7e5bd21d1.json","parameterValues":[]}